oak_attestation_verification/tests/verifier_tests.rs

```diff
@@ -119,9 +125,8 @@ fn create_reference_values() -> ReferenceValues {
         ..Default::default()
     };
     let kernel_layer = KernelLayerReferenceValues {
-        kernel_image: Some(skip.clone()),
+        kernel: Some(kskip.clone()),
```

Comment:
I don't think you need to clone here since it is only used once.

Reply:
Done.